src/main/java/lotto/exception/LottoPurchaseAmountException.java

```diff
@@ -3,10 +3,10 @@
 /*
 사용자가 로또 구입 금액을 잘못 입력 했을때 발생하는 예외.
  */
-public class InvalidLottoPurchaseAmountException extends RuntimeException{
+public class LottoPurchaseAmountException extends RuntimeException{
```

Comment:
```suggestion
public class LottoPurchaseAmountException extends RuntimeException {
```
포맷팅하는 습관을 만드시거나 자동으로 포맷팅하는 방법을 찾아보시면 좋을 것 같습니다 :)